### DISCUSSION TITLE
[Enhancement] Direct Sound output for listening in 5.1 setups

### DISCUSSION THREAD

**Discussion by niyonsv** [ciderapp/Cider#671] [Feature Request]
**Is your feature request related to a problem? Please describe.**
My PC has Intel Motherboard's Realtek audio, and the built-in 5.1 setup is used as output. Also I use Equalizer APO with Peace s/w to configure and customize the audio experiences. But with the Cider (and also with the AME) the sound comes only from the Left and Right channels instead of 5.1 setup.

When I'm using the iTunes for Windows official s/w there's and option to choose the output (see attached image - the options is inside the 'Play Audio Using' dropdown) and to listen in 5.1 with iTunes I chose the Direct Sound instead of Windows Audio Session. If I select the latter option the sound comes from L and R channels like when using Cider.

![itunes-windows-playback-preferences](https://user-images.githubusercontent.com/14888256/148513240-de6f4210-fb25-4993-b49f-1aff6caad701.PNG)

**Describe the solution you'd like**
So if there's an option in Cider to choose the Direct Sound output it'd be super useful.

**Comment by Alvyssy** [comment]
Add Dolby Atmos if it's possible on the way please, I would really love it!

**Comment by Nosh118** [comment]
Hijacking this to also ask for support for WASAPI and WASAPI exclusive

**Reply by neeeeeeeeeeee** [reply]
I want WASAPI too, but the tune quality from the cider... I dont think decoding device can save it... 🥲

**Comment by lance-374** [comment]
> Add Dolby Atmos if it's possible on the way please, I would really love it!

Adding Dolby Atmos or lossless is currently impossible because Apple doesn't provide an API for it.

**Comment by Alvyssy** [comment]
> > Add Dolby Atmos if it's possible on the way please, I would really love it!
> 
> Adding Dolby Atmos or lossless is currently impossible because Apple doesn't provide an API for it.

Sorry for the dumb question in advance but can the Android App be reverse engineered to see how Atmos is done?

**Reply by GamingLiamStudios** [reply]
I am actually doing that rn lol. Going very slow tho

**Reply by BlackwingTech** [reply]
Hey, has anything worked out lately to reverse engineer the entire application, the dolby atmos specifically version dax2.3.1.30_r1?
